src/openstudio_lib/OSCollapsibleItemList.cpp

```diff
@@ -62,6 +62,7 @@ OSCollapsibleItemList::OSCollapsibleItemList(bool addScrollArea, QWidget* parent
   this->setLayout(outerVLayout);
 
   m_searchBox = new QLineEdit();
+  m_searchBox->setClearButtonEnabled(true);
```

Comment:
Lol, that icon is a broom?

Reply:
Hehe, weird on Windows it is a nice x button.